src/rubislices.py

```diff
@@ -41,6 +41,10 @@ def sub_run(self, body):
         )
         clean_up()
 
+    def parse_body_and_run(self, body):
+        self.body['content'] = run_summariser_censor(body['content'])
```

Comment:
Would it be better if we use a more descriptive name using verbs like summarise_and_censor instead?

Reply:
Also just realised that the code at line 45 can actually be added to the sub_run function itself, instead of creating a new function then running like 45 and then calling sub_run again. Do you think we should change the code as such?

Reply:
True, will do!